packages/gatsby-source-prismic/src/create-schema-customization.ts

```diff
@@ -53,11 +49,10 @@ export const createBaseTypes: RTE.ReaderTaskEither<
       RTE.sequenceArray,
     ),
   ),
-  RTE.bind('imgixTypes', () => buildImgixImageTypes),
   RTE.map((scope) =>
     GatsbyGraphQLTypeM.concat(
       scope.baseTypes as Mutable<typeof scope.baseTypes>,
-      scope.imgixTypes,
+      [],
```

Comment:
If the Imgix types end up not being created here, the whole `RTE.map` call on line 52 can be removed. The `createTypes` call below will need to be modified to accept the `baseTypes` scope value.

(I can handle this if it's not clear what I mean)

Reply:
I did actually try to do this but I wasn't too familiar with `RTE`. 

I understand what you mean but I'm not sure how to do it - could you drop a code suggestion here and I'll accept it?

Reply:
Sure thing, it could be rewritten to look like this:

```typescript
export const createBaseTypes: RTE.ReaderTaskEither<
  Dependencies,
  never,
  void
> = pipe(
  [
    buildEmbedType,
    buildGeoPointType,
    buildImageDimensionsType,
    buildImageThumbnailType,
    buildLinkType,
    buildLinkTypeEnumType,
    buildSliceInterface,
    buildStructuredTextType,
    buildTypePathType,
  ],
  RTE.sequenceArray,
  RTE.chain((types) => createTypes(types as Mutable<typeof types>)),
  RTE.map(constVoid),
)
```

Since the array no longer needs to be merged with the Imgix types, it could be simplified into just a list of types being passed into the `createTypes` function. I've found `RTE` to be pretty powerful to work with and organize code, so let me know if you want to learn more!

Reply:
I tried doing the code suggestion thing but couldn't get it to edit the rest of the function. Sorry!

Reply:
Thanks for doing this, even though it turns out since we're using `buildImgixImageTypes` again we won't use this (as you said)